polymer.json

```diff
@@ -1,7 +1,7 @@
 {
   "lint": {
     "rules": ["polymer-3"],
-    "filesToIgnore": ["**/node_modules/**/*.js"],
+    "filesToIgnore": ["**/node_modules/**/*.js", "**/ui/viewers/pdfjs/**/**/*.js"],
```

Comment:
@Gabez0r tried different paths here, but if it starts with `ui/...` the files are not ignored 🤔

Reply:
Does it work with `"**/ui/viewers/pdfjs/**/**/*.js"`?

Reply:
If it does, I'm fine with it!

Reply:
Yes, everything is working with `**/ui/viewers/pdfjs/**/**/*.js`.